pkg/controller/certificatesigningrequest/ipsec_csr_approver.go

```diff
@@ -54,6 +49,21 @@ var ipsecTunnelUsages = sets.New[string](
 
 var _ approver = (*ipsecCSRApprover)(nil)
 
+func getAntreaAgentServiceAccount() string {
+	agentServiceAccountName := strings.Join([]string{
+		"system", "serviceaccount", env.GetAntreaNamespace(), "antrea-agent",
+	}, ":")
+
+	return agentServiceAccountName
```

Comment:
nit: it would be more concise not to define an extra variable. You can just `return `strings.Join(...)`

Reply:
oh I get it